source/shared/cpp/ObjectModel/ParseUtil.h

```diff
@@ -99,27 +102,29 @@ namespace AdaptiveSharedNamespace
 
         template<typename T>
         std::vector<std::shared_ptr<T>> GetElementCollectionOfSingleType(
-            ParseContext& context,
+            ParseContext & context,
```

Comment:
NIT: difference in formatting the dereference operator here from elsewhere in the file...is this on purpose?

Reply:
hmm... we usually lean on `clang-format` to do this for us... maybe it didn't apply here? I'll fix up :)

Reply:
fixed in 2a01ac6